packages/ember-toucan-core/package.json

```diff
@@ -72,6 +73,8 @@
     "@types/ember__debug": "^4.0.0",
     "@types/ember__engine": "^4.0.0",
     "@types/ember__error": "^4.0.0",
+    "@types/ember__helper": "^4.0.2",
+    "@types/ember__modifier": "^4.0.3",
```

Comment:
With `gjs/gts` we need to import helpers like `on`, `fn`, and `hash` explicitly.  By adding these dependencies, it allows us to do so (see below!)

Reply:
Being able to trace helpers back to `import` statements is perhaps my favorite part of `gjs/gts`.

Reply:
I 100% agree 🎉